ajax/libs/Flux-Slider/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Flux-Slider",
+  "version": "1.4.4",
+  "filename": "js/flux.min.js",
+  "description": "CSS3 Animation based image transitions",
+  "homepage": "http://www.joelambert.co.uk/flux/",
+  "keywords": [
+    "css3",
+    "animation",
+    "image",
+    "transitions",
+    "plugin"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/joelambert/Flux-Slider"
+  },
+  "license": "MIT",
+  "author": "Joe Lambert <@joelambert> (http://www.joelambert.co.uk/)",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/joelambert/Flux-Slider.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "js/flux*",
+          "css/style.css"
```

Comment:
Let's remove `"css/style.css"`
Because this file is used by the library website, only.
Thank you.